cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,53 @@
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
+import CartPage from '../support/pages/cartPage.pageObject';
+import { faker } from '@faker-js/faker';
+
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+const homePage = new HomeAndCataloguePageObject();
+const cartPage = new CartPage();
 
+describe('Checkout process', () => {
+  before(() => {
+    homePage.visit();
   });
 
-  it('', () => {
+  it('should complete the checkout process', () => {
+    const orderDetails = {
+      name: faker.name.firstName(),
+      country: faker.address.country(),
+      city: faker.address.city(),
+      card: faker.finance.creditCardNumber(),
+      month: faker.date.month(),
+      year: faker.date.future().getFullYear().toString()
+    };
+
+    homePage.visit();
+
+    homePage.clickOnCategory('Laptops');
+
+    homePage.clickOnProduct('Sony vaio i7');
+
+    cy.contains('a', 'Add to cart').click();
+
+    cy.on('window:alert', (text) => {
+      expect(text).to.contains('Product added');
+    });
+
+    cy.get('#cartur').click();
```

Comment:
```suggestion
    cy.contains('a', 'Add to cart').click();

    cy.on('window:alert', (text) => {
      expect(text).to.contains('Product added');
    });

    cy.get('#cartur').click();
```
wrap this lines of code with methods in pageObjects